components/molecules/AuthContentWrapper/auth-content-wrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import clsx from "clsx";
+
+interface AuthContentWrapperProps {
+  isAuthenticated: boolean;
+  children: React.ReactNode;
+}
+
+const AuthContentWrapper: React.FC<AuthContentWrapperProps> = ({ isAuthenticated, children }) => {
```

Comment:
Kindly note, the name is very much a Draft!

Unless y'all think this is just fine.